solidity/dashboard/src/actions/web3.js

```diff
@@ -176,3 +176,30 @@ export const withdrawGroupMemberRewards = (
     meta,
   }
 }
+
+export const withdrawAllLiquidityRewards = (
+  liquidityPairContractName,
+  meta
+) => {
+  return {
+    type: WEB3_SEND_TRANSACTION,
+    payload: {
+      contractName: liquidityPairContractName,
+      methodName: "exit",
+      args: [],
```

Comment:
Let's skip `args` param because the [createTransactionEventChannel](https://github.com/keep-network/keep-core/blob/master/solidity/dashboard/src/sagas/web3.js#L15) function sets `args` param to an empty array by default.